beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -1516,6 +1517,47 @@ impl<AppReqId: ReqId, TSpec: EthSpec> Network<AppReqId, TSpec> {
         }
     }
 
+    fn parse_upnp_event(&mut self, event: libp2p_upnp::Event) {
```

Comment:
Since the existing methods for handling events are named with the prefix `inject_`, perhaps we should align with this naming convention? For example, we could name it `inject_upnp_event`.

Reply:
yeah makes sense. Updated, thanks!